src/webui/static/app/codec.js

```diff
@@ -252,9 +253,17 @@ var codec_profile_forms = {
                     bit_rate_scale_factor_field.setDisabled(false);
                     qp_field.setDisabled(false);
                     low_power_field.setDisabled(false);
+                    desired_b_depth_field.setDisabled(false);
                     break;
                 case 1:
                     // Intel
+                    var low_power = low_power_field.getValue();
+                    if (low_power == 1) {
+                        desired_b_depth_field.setDisabled(true);
```

Comment:
Don't use `if` here, just add the condition directly instead of true/false

Reply:
done